activejob/lib/active_job/railtie.rb

```diff
@@ -72,7 +72,7 @@ class Railtie < Rails::Railtie # :nodoc:
         app.config.active_job.log_query_tags_around_perform
 
       if query_logs_tags_enabled
-        app.config.active_record.query_log_tags << :job
+        app.config.active_record.query_log_tags << :job unless app.config.active_record.query_log_tags.include?(:job)
```

Comment:
```suggestion
        app.config.active_record.query_log_tags |= [:job]
```

Might as well simplify here too.

Reply:
Ahh, I didn't want to initialize the extra array, but I guess that's pretty trivial in the scheme of things. Thanks!